src/styles/filenameinput.styl

```diff
@@ -1,5 +1,8 @@
 @require 'cozy-ui'
 
 .fil-file-name-input
-    @extends $form
     @extends $form-text
+
+    input[type=text]
+        padded(6px)
```

Comment:
Could you please write (I know i'm a little monomaniac on this…):
`padded 6px` instead of `padded(6px)`
It will do exactly the same thing but values would be aligned. 😄